openjdk.test.jck/src/test.jck/net/adoptopenjdk/stf/Jck.java

```diff
@@ -394,12 +394,7 @@ public void execute(StfCoreExtension test) throws Exception {
 		}
 	}
 
-	
-	public void tearDown(StfCoreExtension test) throws Exception {
-		//test.doEchoFile("JCK results provided by Test harness", head -5 test.env().getResultsDir().childDirectory("workdir/jtData").childFile("harness.trace"));
-		test.doEchoFile("Test run Summary:\n", test.env().getResultsDir().childDirectory("report/text").childFile("summary.txt"));
-		test.doEchoFile("Failed Tests are:\n", test.env().getResultsDir().childDirectory("report/html").childFile("failed.html"));
-	}
+	public void tearDown(StfCoreExtension test) throws Exception {}
```

Comment:
Does this information is duplicate if we have xml report? If not maybe keep it.

Reply:
When we use XML style jck output those child files "summary.txt" and "failed.html" are not generated. Those are only generated when report type is default, e.g. HTML. And since we are using more elaborate JUNIT style reporting in Jenkins now, we no longer need them either.